### ISSUE TITLE
Setting default permission to true is doing nothing.

### ISSUE THREAD

**Issue by gbrant** [PEXPlugins/PermissionsEx#2264]
I'm an admin for my server, I've been trying since I installed sponge to give myself a default permission of true, tried a whole bunch of different ways to more or less give myself access to all commands, and none of them work.

Doing
/pex user tooforgetful def true
does absolutely nothing, and still has plenty of commands locked out for whatever reason.

Creating an admin group with default permissions and adding myself to it has the same effect, sill not giving me access and making me run some commands through the console (some of which cannot work properly through it, making them effectively worthless)

**Comment by Stormbow** [comment]
Serious question: Where did you learn that was a command?

**Comment by zml2008** [comment]
Yeah, this is PEX for Sponge that there's an issue with here.

The only thing I can think of being an issue here is that you ran this command before your user had logged in -- when PEX2 does uuid lookups it only uses the local cache since command argument handling is synchronous and we don't want to block the main thread.

Check your permissions.json to confirm if the default was set under your uuid. If it wasn't, delete the existing entry and after having logged in to your server at least once set the default again (confirming that it's stored by UUID).

If that doesn't work, we'll have to investigate further.

**Comment by gbrant** [comment]
It's in the sponge list of commands, and is typically used for assigning default values to groups.
For example, doing 

/pex group Default def false 

Will set the permissions for every command for the group Default to false, meaning nobody in the Default group can use commands, unless you specifically set them. You can see it with a couple more examples over in this thread over here -> https://forums.spongepowered.org/t/guide-pex-quick-command-reference/8738

As far as the issue itself though, this seems to be an issue with if a user is in a group that has a default param of false. I ended up completely wiping the permissions.json file and starting over from scratch, and making an admin group which defaulted to true for all commands and then adding myself to it seemed to work just fine, but hadn't before when the admin group was created after a default with a false default.

Long story short, it seems like it'll work unless there's a conflicting global default somewhere else, even if you specifically set it different for the user.

**Comment by zml2008** [comment]
Yeah, negative overrides positive of the same weight I guess

What you could do for yourself is /pex user someone def 2 

(Permissions are set with int-weighted values -- positive evaluate to true, negative to false, 0 to check parent, and higher numbers override lower ones)

**Comment by Stormbow** [comment]
Ahhh! I was looking in the wrong place for commands.  Silly me.
